pkg/builtinchecks/yamls/pdbs-max-unavailable.yaml

```
@@ -0,0 +1,7 @@
+name: "pdb-max-unavailable"
+description: "Indicates when a DeploymentLike object has a \"Pod Disruption Budget\" specified with a MaxUnavailable value that is too restrictive."
+remediation: "Change the \"Pod Disruption Budget\" to have a \"MaxUnavailableValue\" set greater than 0. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```

Would you agree that "prevents disruptions" is more informative than "too restrictive"?

Also adjusting mark-up to be consistent with other checks (they don't seem to denote literal object and field names typographically) and fixing the "MaxUnavailableValue" typo.

```suggestion
description: "Indicates when a deployment-like object has a PodDisruptionBudget specified with a maxUnavailable value that prevents disruptions."
remediation: "Change the PodDisruptionBudget to have maxUnavailable set to a value greater than 0. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```

The description still sounds as if the check reported deployment-like objects rather than PDBs. How about this?

```suggestion
description: "Indicates when a PodDisruptionBudget has a maxUnavailable value that prevents disruptions of deployment-like objects it is applied to."
remediation: "Change the PodDisruptionBudget to have maxUnavailable set to a value greater than 0. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```